plugins/gitlab/src/org/jetbrains/plugins/gitlab/api/GitLabServerPath.kt

```diff
@@ -34,8 +34,13 @@ class GitLabServerPath : ServerPath {
   val restApiUri: URI
     get() = toURI().resolveRelative("api/v4/")
 
+  /**
+   * I think the first part of this condition was wrong and always false - since the default String uri is always https://gitlab.com and so it's
+   *  never start with gitlab.com
+   *
+   */
   val isDefault: Boolean
-    get() = uri.startsWith("gitlab.com", true) || uri.contains("/gitlab.com", true)
+    get() = uri.startsWith("https://gitlab.com", true) || uri.contains("/gitlab.com", true)
```

Comment:
I would say that it would be better to strip the schema from `uri` for this comparison. One can specify schema as `http`

Reply:
I think I can do this during the "merge", so it's fine